config/projects.yml

```diff
@@ -5,7 +5,7 @@ admin-bundle:
       versions:
         symfony: ['3.4']
         sonata_core: ['3']
-        sonata_block: ['3']
+        #sonata_block: ['3']
```

Comment:
Sorry, but I won't merge commented code/config.

If this one is not compatible with block-bundle 3, why not putting v4/dev-master?

Reply:
When `sonata_block: ['3']` in present in `config/projects.yml` for some bundle, these lines are added to `.travis-ci.yml`:

```yaml
matrix:
  include:
    - php: '7.2'
      env: SONATA_BLOCK=3.*
    - php: '7.2'
      env: SONATA_BLOCK='dev-master as 3.x-dev'
  allow_failures:
    - env: SONATA_BLOCK='dev-master as 3.x-dev'
```

I cannot put `v4/dev-master` because after https://github.com/sonata-project/SonataAdminBundle/pull/4938 `master` branch of AdminBundle will become incompatible with current stable BlockBundle. And [travis job](https://travis-ci.org/sonata-project/SonataAdminBundle/jobs/357250777) for `SONATA_BLOCK=3.*` will always fail for `master` branch.

Also, all bundles, that uses BlockBundle are already use `3.x` version, so i guess there is no sence in additional job with `SONATA_BLOCK=3.*`

Also current stable branches of all bundles are already incompatible with `master` branch of BlockBundle and job with `SONATA_BLOCK='dev-master as 3.x-dev'` is always failing (but it is allowed).

So, with my PR i tried to remove jobs with `SONATA_BLOCK` env var